services/core/shared/core/compat/sdk_v5/blockchainIndex.js

```diff
@@ -102,6 +106,10 @@ const getIndexVerifiedHeight = () => blockchainStore.get('indexVerifiedHeight');
 const setIndexDiff = (height) => blockchainStore.set('indexStatus', height);
 const getIndexDiff = () => blockchainStore.get('indexStatus');
 
+// Key-based account update
+// There is a bug that does not update public keys
+const KEY_BASED_ACCOUNT_UPDATE = false;
```

Comment:
Now that the public keys are getting indexed, do we want to enable this?

Reply:
I would prefer to enable it in a different PR.